Cargo.toml

```diff
@@ -110,18 +110,18 @@ bevy = { version = "0.10", features = ["mp3"] }
 bincode = "2.0.0-rc.3"
 chrono = "0.4.24"
 clap = { version = "4.0", features = ["derive"] }
-criterion = "0.4"
+criterion = "0.5.1"
 dirs = "5.0.0"
 enum-iterator = "1.4.0"
 enum-map = "2.3.0"
-fastrand = "1.9.0"
```

Comment:
`futures-lite` require `^1.3.4` so I would not change this dependency until `futures-lite` upgrade.